web/index.html

```diff
@@ -382,13 +383,13 @@
       </header>
       <div id="layer-modal-main">
         <div id="layer-categories">
-          
+
         </div>
         <div id="selected-category">
-          
+
```

Comment:
Ah, the differences is that there were spaces after the line return that are no longer present.